### DISCUSSION TITLE
Can Nuxt3 provide a Socket.io server?

### DISCUSSION THREAD

**Discussion by borsTiHD** [nuxt/nuxt#17275] [Questions]
Hi all,

is ist possible with `~/server/api` or `~/server/routes` to provide a socket.io server?

I haven't tried anything in that direction yet, but would hope for an easier setup and use, especially with the later deploy at a node.js hoster (e.g. Vercel). I would like to develop a small card game for practice in which several people can participate. For this I would find socket.io perfect for communication among each other.

As an alternative I only know the way to set up an own Express server with socket.io. However, I would like to avoid this to make the deploy via Vercel as simple as possible and also to avoid things like CORS.

Many thanks in advance.

**Comment by borsTiHD** [comment]
I found out about https://github.com/nuxt/framework/discussions/1189
However, it was mentioned there that this would not work with Vercel, is that correct?

**Reply by wobsoriano** [reply]
Vercel is serverless. Yes, it won't work.

**Reply by borsTiHD** [reply]
Sorry for the late response and thanks. :)
